agent/helpers.go

```diff
@@ -28,14 +28,17 @@ import (
 	"sync"
 	"syscall"
 	"time"
+
+	utilexec "github.com/romana/core/pkg/util/exec"
+	utilos "github.com/romana/core/pkg/util/os"
 )
 
 // NewAgentHelper returns Helper with initialized default implementations
 // for all interfaces.
 func NewAgentHelper(agent *Agent) Helper {
 	helper := new(Helper)
-	helper.Executor = new(DefaultExecutor)
-	helper.OS = new(DefaultOS)
+	helper.Executor = new(utilexec.DefaultExecutor)
+	helper.OS = new(utilos.DefaultOS)
```

Comment:
OS as in "operating system" but also as a name of module we are wrapping - "os" from standard library.
Now, while this component was a part of agent we needed a suffix after "Default" which is now obsolete - and should be "utilsos.Default" and "utilexec.Default". 

This current change, however, only about packaging. I deliberately skipped naming issue in order to keep this commit small. Naming need to be addressed but not in this change.